pages/common/micromamba.md

```diff
@@ -0,0 +1,37 @@
+# micromamba
+
+> A fast, minimal, standalone package and environment manager for conda packages.
+> Drop-in replacement for conda, ideal for CI, Docker, and lightweight setups.
+> More information: <https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html>.
+
+- Create a new environment at a specific path, installing named packages into it:
+
+`micromamba create -p {{/path/to/env}} {{python=3.11 numpy}}`
```

Comment:
```suggestion
`micromamba create {{[-p|--prefix]}} {{/path/to/env}} {{python=3.11 numpy}}`
```
Either use only longform options or option placeholders. Only shortform is not advised